compilator/compilator.go

```diff
@@ -471,16 +470,21 @@ func (c *Compilator) compilePackage(pkg *model.Package) (err error) {
 			return color.GreenString("compilation-%s > %s", color.MagentaString("%s", pkg.Name), color.RedString("%s", line))
 		},
 	)
-	exitCode, container, err := c.DockerManager.RunInContainer(
-		containerName,
-		c.BaseImageName(),
-		[]string{"bash", containerScriptPath, pkg.Name, pkg.Version},
-		c.getTargetPackageSourcesDir(pkg),
-		c.getPackageCompiledTempDir(pkg),
-		c.keepContainer,
-		stdoutWriter,
-		stderrWriter,
-	)
+	mounts := map[string]string{
+		c.getTargetPackageSourcesDir(pkg): docker.ContainerInPath,
+		c.getPackageCompiledTempDir(pkg):  docker.ContainerOutPath,
+		"source_mount":                    "/var/vcap/source/",
```

Comment:
how does docker know it's a "temporary" volume?
Because it's not an absolute path?
If so, can you please comment the code with that info?
Also, UUID sounds good.